### PR TITLE
Feta/autorenew subtree

### PR DESCRIPTION
**Pull request by marie-j**
Subtree for original PR : https://github.com/ovh-ux/ovh-manager-dedicated/pull/1421
ref: MBE-397, MANAGER-3264

### Requirements
- [x] 2API
- [x] https://github.com/ovh/manager/pull/1468

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#6710.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_us#6710.1 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_eu#6710.1 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_us#6710.3 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_eu#6716.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_us#6716.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_us#6716.2 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_eu#6716.1 ✘  
* deploy_uxci
  * deploy_uxci ✘

File: packages/manager/apps/dedicated/client/app/billing/autoRenew/autorenew.service.js [hunk 1]
```diff
@@ -49,7 +54,7 @@ export default class {
       .query().$promise;
   }
 
-  getServices(count, offset, search, type, renewDateType, status, order, nicBilling) {
+  getServices(count, offset, search, type, renewDateType, status, state, order, nicBilling) {
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/billing/autoRenew/autorenew.service.js]: Hmm no eslint error for line-length (I know the line length was already above the limit) ?

Comment (marie-j) [reply packages/manager/apps/dedicated/client/app/billing/autoRenew/autorenew.service.js]: No I didn't get an error. I will check my config though

**Pull request merged**